src/Runner/Runner.php

```diff
@@ -222,7 +230,10 @@ private function fixParallel(): array
 
         $processesToSpawn = min(
             $this->parallelConfig->getMaxProcesses(),
-            (int) ceil($this->fileCount / $this->parallelConfig->getFilesPerProcess())
+            max(
+                1,
```

Comment:
`$this->fileCount` may be empty, and then we do not even run `fixParallel()`

but also `$this->fileCount` may not be empty, but after filtering out cached files etc, we have no files to be run. with that, i put another guard here with `max(1, ...)`